DuckDuckGo/NetworkProtectionAccessController.swift

```diff
@@ -54,12 +42,10 @@ protocol NetworkProtectionAccess {
 struct NetworkProtectionAccessController: NetworkProtectionAccess {
```

Comment:
I think we may be able to get rid of this whole class, but we can do this in a followup.